terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,92 @@
+  oauth2_client_secret      = local.service_secrets["oauth2_client_secret"]
+
+  # create a map of secret name => secret arn to pass into ecs service module
+  # using the trimprefix function to remove the prefixed path from the secret name
+  secrets_arn_map = {
+    for sec in data.aws_ssm_parameter.secret:
+      trimprefix(sec.name, "/${local.name_prefix}/") => sec.arn
+  }
+
+  service_secrets_arn_map = {
+    for sec in module.secrets.secrets:
+      trimprefix(sec.name, "/${local.service_name}-${var.environment}/") => sec.arn
+  }
+
+  # TODO: task_secrets don't seem to correspond with 'parameter_store_secrets'. What is the difference?
+  task_secrets = [
+    { "name": "COOKIE_SECRET", "valueFrom": "${local.secrets_arn_map.web-oauth2-cookie-secret}" },
+    { "name": "CHS_API_KEY", "valueFrom": "${local.service_secrets_arn_map.chs_api_key}" },
+    { "name": "CACHE_SERVER", "valueFrom": "${local.service_secrets_arn_map.cache_server}" },
+    { "name": "OAUTH2_CLIENT_ID", "valueFrom": "${local.service_secrets_arn_map.oauth2_client_id}" },  
+    { "name": "OAUTH2_CLIENT_SECRET", "valueFrom": "${local.service_secrets_arn_map.oauth2_client_secret}" },
+    { "name": "ACCOUNT_URL", "valueFrom": "${local.service_secrets_arn_map.account_url}" },
+    { "name": "INTERNAL_API_URL", "valueFrom": "${local.service_secrets_arn_map.internal.api.url}"},
+  ]
+
+  task_environment = [
+    { "name": "NODE_PORT", "value": "${local.container_port}" },
+    { "name": "API_LOCAL_URL", "valueFrom": "${var.api.local.url}"},
+    { "name": "AUTHENTICATION_MIDDLEWARE", "valueFrom": "${var.authentication.middleware}"},
+    { "name": "CACHE_SERVER", "valueFrom": "${var.cache.server}"},
+    { "name": "CDN_HOST", "valueFrom": "${var.cdn_host}"},
+    { "name": "CHS_API_KEY", "valueFrom": "${var.chs.api.key}"},
+    { "name": "CHS_URL", "valueFrom": "${var.chs_url}"},
+    { "name": "COOKIE_DOMAIN", "valueFrom": "${var.cookie_domain}"},
+    { "name": "COOKIE_EXPIRATION_IN_SECONDS", "valueFrom": "${var.expiration.in.seconds}"},
+    { "name": "COOKIE_NAME", "valueFrom": "${var.cookie_name}"},
+    { "name": "COOKIE_SECRET", "valueFrom": "${var.cookie.secret}"},
+    { "name": "COOKIE_SECURE_FLAG", "valueFrom": "${var.cookie.secret.flag}"},
+    { "name": "DEFAULT_SESSION_EXPIRATION", "valueFrom": "${var.default.session.expiration}"},
+    { "name": "EXTENSIONS_API_URL", "valueFrom": "${var.extensions.api.url}"},
+    { "name": "EXTENSIONS_PROCESSOR_API_URL", "valueFrom": "${var.extensions.processor.api.url}"},
+    { "name": "FEATURE_REQUEST_COUNT", "valueFrom": "${var.feature.request.count}"},
+    { "name": "HUMAN_LOG", "valueFrom": "${var.human.log}"},
+    { "name": "LOG_LEVEL", "valueFrom": "${var.log_level}"},
+    { "name": "MAXIMUM_EXTENSION_REQUESTS_PER_DAY", "valueFrom": "${var.maximum.extension.requests.per.day}"},
+    { "name": "MAX_EXTENSION_PERIOD_IN_MONTHS", "valueFrom": "${var.max.extension.period.in.minutes}"},
```

Comment:
var name should be in months not minutes for MAX_EXTENSION_PERIOD_IN_MONTHS

Reply:
months not minutes